sdk/webpubsub/azure-messaging-webpubsub/swagger/README.md

```diff
@@ -18,6 +18,8 @@ use-iterable: true
 data-plane: true
 generate-sync-async-clients: true
 service-name: WebPubSubService
+generate-builder-per-client: false
 service-versions:
-  - '2021-10-01'
+  - '2021_10_01'
```

Comment:
why the difference?

Reply:
good catch, it's a mistake when adding back previous service version. I am updating.